subprojects/composite-builds/composite-builds.gradle.kts

```diff
@@ -40,3 +40,9 @@ testFixtures {
 testFilesCleanup {
     policy.set(WhenNotEmpty.REPORT)
 }
+
+tasks.withType<Test> {
+    jvmArgs("-javaagent:${rootProject.file("file-leak-detector-1.11-jar-with-dependencies.jar=trace").getAbsolutePath().replace("\\", "/")}")
```

Comment:
As discussed, that is probably the wrong place to attach the javaagent: the Gradle process runs in a different JVM from the test task.

Reply:
Yes. After configuring it correctly, I probably found a bug in Groovy, but this is out of this PR scope.